docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2858,6 +2861,29 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+
+  EventAuthField:
+    type: object
+    description: |
+      Parameters to define per-event authorization. This field can be used with an authorization plugin
+      to differentiate which authorized consumers for an event type can receive an event published to
+      the event type. In case the field is null or not present in the event, it will be allowed to be read
+      by all authorized consumers.
+      Once defined, the definition for EventAuthField for an event-type cannot be updated or removed.
+    properties:
+      path:
+        type: string
+        description: |
+          Path to a string field in the event that contains the a classifier, which can be used by an
```

Comment:
`the a`

Reply:
Also it's important to define that a path is in dot notation. You can give an example as well to make it clearer.